src/app/shared/podcast-nav/podcast-nav-list.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { PodcastModel } from '../../ngrx';
+
+@Component({
+  selector: 'metrics-podcasts-nav-list',
```

Comment:
Plural "podcasts" vs singular "podcast" in the component name?

Reply:
oops! missed a spot in the rename to podcast-no-s-mav, will update!